cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -206,6 +207,43 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
     )
   }
 
+  @Test def create_text_index() {
+    val textProvider = TextIndexProviderFactory.DESCRIPTOR.name()
+
+    testQuery(
+      title = "Create a node text index",
+      text = "A named text index on a single property for all nodes that have a particular label can be created with `CREATE TEXT INDEX index_name FOR (n:Label) ON (n.property)`. " +
+        "Note that the index is not immediately available, but is created in the background.",
+      queryText = "CREATE TEXT INDEX node_index_name FOR (n:Person) ON (n.surname)",
+      optionalResultExplanation = "Note that text indexes do not support multiple properties and that the index name needs to be unique.",
```

Comment:
```suggestion
      optionalResultExplanation = "Note that text indexes do not support multiple properties and that the index name must be unique.",
```

Reply:
This I think is everywhere and not just here but I'll check